eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -40,8 +40,61 @@ jobs:
     BuildInParallel: ${{ parameters.BuildInParallel }}
     TimeoutInMinutes: ${{ parameters.TimeoutInMinutes }}
     Location: ${{ parameters.Location }}
-    SubscriptionConfiguration: ${{ parameters.SubscriptionConfiguration }}
     ServiceDirectory: ${{ parameters.ServiceDirectory }}
+    CloudConfigurations:
+      PublicCloud:
+        SubscriptionConfiguration: ${{ parameters.SubscriptionConfiguration }}
+        ArmTemplateParameters: >-
+          @{ keyVaultDomainSuffix = '.vault.azure.net';
+          storageEndpointSuffix = 'core.windows.net';
+          endpointSuffix = '.cognitiveservices.azure.com';
+          azureAuthorityHost = 'https://login.microsoftonline.com/';
+          keyVaultEndpointSuffix = '.vault.azure.net'
+          }
+        EnvVars:
+          SERVICE_MANAGEMENT_URL: https://management.core.windows.net/
+          STORAGE_ENDPOINT_SUFFIX: core.windows.net
+          RESOURCE_MANAGER_URL: https://management.azure.com/
+          SEARCH_ENDPOINT_SUFFIX: search.windows.net
+          COSMOS_TABLES_ENDPOINT_SUFFIX: cosmos.azure.com
+      ${{ if and(eq(variables['System.TeamProject'], 'internal'), endsWith(variables['Build.DefinitionName'], ' - tests-weekly')) }}:
```

Comment:
This is an good starting point and will enable our scheduled runs but we might still need to figure out how we can allow people to trigger a test run for a particular cloud.